mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageIdToImapUidDAOTest.java

```diff
@@ -46,12 +46,14 @@
 
 import reactor.core.publisher.Flux;
 
-@ExtendWith(CassandraRestartExtension.class)
 class CassandraMessageIdToImapUidDAOTest {
     public static final CassandraModule MODULE = CassandraModule.aggregateModules(
             CassandraSchemaVersionModule.MODULE,
             CassandraMessageModule.MODULE);
 
+    @RegisterExtension
+    static CassandraRestartExtension cassandraRestartExtension = new CassandraRestartExtension().forceRestart();
```

Comment:
why it would not work with allowRestart ?

Reply:
Because we want to force a restart on this particular class. If it's just allowRestart, it might not restart at all depending on the state of the counter at that moment. 

The problem as stated above is that still often on the CI build you have the error posted as the first comment. We noticed that this error was not really there anymore when we disabled the test forks and it came back when we re-enabled it. Thus, the point of this PR of forcing a restart before that test class is executed. 

I don't say it's THE solution, but just trying a new strategy here